### ISSUE TITLE
OSError: sndfile library not found during docs build

### ISSUE THREAD

**Issue by lostanlen** [mir-dataset-loaders/mirdata#192]
Spotted by @r4b1tt on latest master today. Unintended consequence of #183, which went from `librosa<0.7.0` to `librosa>=0.7.0`

Possibly related: https://github.com/librosa/librosa/issues/1017

Traceback below:

```python
mirdata.gtzan_genre
^^^^^^^^^^^^^^^
WARNING: autodoc: failed to import module 'gtzan_genre' from module 'mirdata'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/sphinx/ext/autodoc/importer.py", line 154, in import_module
    __import__(modname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/checkouts/latest/mirdata/gtzan_genre.py", line 18, in <module>
    import librosa
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/librosa/__init__.py", line 12, in <module>
    from . import core
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/librosa/core/__init__.py", line 126, in <module>
    from .audio import *  # pylint: disable=wildcard-import
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/librosa/core/audio.py", line 10, in <module>
    import soundfile as sf
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/soundfile.py", line 142, in <module>
    raise OSError('sndfile library not found')
OSError: sndfile library not found
```

Proposed fix: explicitly add soundfile to docs/requirements.txt

**Comment by rabitt** [comment]
Unfortunately the docs on readthedocs are still not building :( Same error as before.

```
WARNING: autodoc: failed to import module 'beatles' from module 'mirdata'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/sphinx/ext/autodoc/importer.py", line 154, in import_module
    __import__(modname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/checkouts/latest/mirdata/beatles.py", line 20, in <module>
    import librosa
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/librosa/__init__.py", line 12, in <module>
    from . import core
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/librosa/core/__init__.py", line 126, in <module>
    from .audio import *  # pylint: disable=wildcard-import
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/librosa/core/audio.py", line 10, in <module>
    import soundfile as sf
  File "/home/docs/checkouts/readthedocs.org/user_builds/mirdata/envs/latest/lib/python3.7/site-packages/soundfile.py", line 142, in <module>
    raise OSError('sndfile library not found')
OSError: sndfile library not found
```

**Comment by rabitt** [comment]
Fixed in #194
